[DOCSP-41529] cleanup ticket and organization (#65)
mongodb/docs-java-rs: source/connect.txt

   
   try (MongoClient mongoClient = MongoClients.create(uri))

Sharded Cluster
~~~~~~~~~~~~~~~

To connect to a sharded cluster, specify the ``mongos`` instance or
instances to the ``MongoClients.create()`` method. To learn more about
sharded clusters, see :manual:`Sharding </sharding/>` in the Server
manual.

You can connect to a single ``mongos`` instance in the following ways:

> - Specify the hostname and the port in a ``ConnectionString``:

Reviewer: S: introduce the code example. Also, I don't think ConnectionString needs to be monospaced:

```suggestion
- Specify the hostname and the port in a connection string, as shown in the following code:
```

Reviewer: Applies to all (both the introductory phrase and the connection string formatting)

  .. code-block:: java
     

>      MongoClient mongoClient = MongoClients.create( "mongodb://localhost:27017" );

Reviewer: S: use "hostname" and "port" placeholders here:
```suggestion
     MongoClient mongoClient = MongoClients.create( "mongodb://<hostname>:<port>" );
```

- Exclude connection string if the ``mongos`` is running on
  ``localhost:27017``:

  .. code-block:: java
     
     MongoClient mongoClient = MongoClients.create();

You can connect to multiple ``mongos`` instances in the following ways:

- Specify the ``ConnectionString`` to contain their hostnames and ports:

  .. code-block:: java
     

>      MongoClient mongoClient = MongoClients.create("mongodb://host1:27017,host2:27017");

Reviewer: S: similar suggestion as above
```suggestion
     MongoClient mongoClient = MongoClients.create("mongodb://<first hostname>:<first port>,<second hostname>:<second port>");
```
